src/libraries/Microsoft.Extensions.Configuration.Json/tests/ConfigurationBinderTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration.Test;
+using Xunit;
+
+namespace Microsoft.Extensions.Configuration.Json.Tests
+{
+    public class ConfigurationBinderTest
+    {
+        public class EnumConf : Dictionary<TestSettingsEnum, string>
```

Comment:
What's the point of this extra class? Why not just use a `Dictionary<TestSettingsEnum, string>` directly?